zebra/zebra_vrf.c

```diff
@@ -368,10 +372,19 @@ static void zebra_rnhtable_node_cleanup(struct route_table *table,
 static void zebra_vrf_table_create(struct zebra_vrf *zvrf, afi_t afi,
 				   safi_t safi)
 {
+	struct route_node *rn;
+	struct prefix p;
+
 	assert(!zvrf->table[afi][safi]);
 
 	zvrf->table[afi][safi] =
 		zebra_router_get_table(zvrf, zvrf->table_id, afi, safi);
+
+	memset(&p, 0, sizeof(p));
+	p.family = afi2family(afi);
+
+	rn = srcdest_rnode_get(zvrf->table[afi][safi], &p, NULL);
+	zebra_rib_create_dest(rn);
```

Comment:
Hm.  If a default route is added to a table, and then removed, won't this free the rib_dest for the 0/0 entry?

Reply:
look at `rib_can_delete_dest`.  This should prevent the garbage collection from happening, but won't prevent it from being cleaned up on shutdown.

Reply:
Indeed I missed that.